src/Microsoft.ML/Models/OnnxConverter.cs

```diff
@@ -10,16 +10,21 @@ namespace Microsoft.ML.Models
     public sealed partial class OnnxConverter
     {
         /// <summary>
-        ///
-        /// This API converts the model to ONNX format by inspecting the transform pipeline 
+        /// <see href="https://onnx.ai/">ONNX</see> is an intermediate representation format 
+        /// for machine learning models. It is used to make models portable such that you can 
+        /// train a model using a toolkit and run it in another tookit's runtime, for example,
+        /// you can create a model using ML.NET(or any ONNX compatible toolkit), convert it to ONNX and 
+        /// then the ONNX model can be converted into say, CoreML, TensorFlow or WinML model 
+        /// to run on the respective runtime.
+        /// This API converts an ML.NET model to ONNX format by inspecting the transform pipeline 
         /// from the end, checking for components that know how to save themselves as ONNX. 
         /// The first item in the transform pipeline that does not know how to save itself 
         /// as ONNX, is considered the "input" to the ONNX pipeline. (Ideally this would be the 
         /// original loader itself, but this may not be possible if the user used unsavable 
         /// transforms in defining the pipe.) All the columns in the source that are a type the 
         /// ONNX knows how to deal with will be tracked. Intermediate transformations of the 
         /// data appearing as new columns will appear in the output block of the ONNX, with names 
-        /// derived from the corresponding column names. The ONNX json will be serialized to a 
+        /// derived from the corresponding column names. The ONNX JSON will be serialized to a 
         /// path defined through the Json option.
```

Comment:
Json -> JSON

Reply:
Json is the name of the command line option.